autoblocks/_impl/api/models.py

```diff
@@ -46,6 +46,13 @@ class DatasetWithItems(Dataset):
     items: List[DatasetItem]
 
 
+@dataclass
+class ManagedTestCase:
+    body: Dict[str, Any]
+    createdAt: str
+    updatedAt: str
```

Comment:
python is snake_case

but let's just drop these both here and in the endpoint since they're not necessary

Reply:
These changes aren't ready for review yet - will update